homeassistant/components/deconz/config_flow.py

```diff
@@ -95,8 +94,7 @@ def __init__(self):
             self.bridges = []
```

Comment:
Shouldn't this abort with error while doing discovery ?

Reply:
Same comment as above, manual input is presented as a fallback if no bridges are discovered.